runtime-common/stdlib/stream/stream-functions.h

```diff
@@ -0,0 +1,122 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include "runtime-common/core/runtime-core.h"
+#include "runtime-common/stdlib/stream/stream-context.h"
+
+using Stream = mixed;
+
+
+constexpr int64_t STREAM_SET_BLOCKING_OPTION = 0;
```

Comment:
Please, don't forget to to add `inline` to everything you define in header files

Reply:
fix